Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt

```diff
@@ -139,31 +145,46 @@ private fun SurveyResult(result: SurveyState.Result, modifier: Modifier = Modifi
     }
 }
 
+@OptIn(ExperimentalLayout::class)
 @Composable
 private fun SurveyTopAppBar(
-    @StringRes surveyTitle: Int,
+    questionIndex: Int,
+    totalQuestionsCount: Int,
     onBackPressed: () -> Unit
 ) {
-    TopAppBar(
-        title = {
+    Column(modifier = Modifier.fillMaxWidth()) {
```

Comment:
If using ConstraintLayout here, I'd lay out the progress indicator as well and remove the Column